karma.conf.js

```diff
@@ -1,19 +1,24 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
-module.exports = function(config) {
+module.exports = function (config) {
   config.set({
     basePath: '',
-    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    frameworks: ['parallel', 'jasmine', '@angular-devkit/build-angular'],
```

Comment:
This is nice 👍 Does it mean we can run tests in parallel ?

Reply:
I checked that for running test cases of sub projects, it's not running separately. Created 3 parallel instance.
With time we can observe the performance. Currently there is one 1 test case in each sub project so no much impact